.github/actions/build-image-terraform/action.yml

```
@@ -39,9 +39,13 @@ runs:
         find "${{ github.workspace }}/packages" -print -exec touch \{} \;
         ls -al "${{ github.workspace }}/packages/"
 
+    - uses: actions/setup-go@v4
+      with:
+        go-version: '1.20'
     - uses: chainguard-dev/actions/setup-registry@main
       with:
         port: 5000
+        crane-version: tip  # TODO: switch to default latest-release after next ggcr release
```

Once https://github.com/chainguard-dev/actions/pull/253 lands this would be:
```suggestion
        disk: true
```